doc/releases/changelog-dev.md

```diff
@@ -29,6 +29,27 @@
 
 <h3>Improvements</h3>
 
+* The decorator `self_inverses` now supports all Hermitian Gates.
+  [(#1136)](https://github.com/PennyLaneAI/catalyst/pull/1136)
+
+  The full list of supported gates are as follows:
```

Comment:
Just remembered this warning in the docs need to be updated to identify all the supported gates (or removed altogether)

https://github.com/PennyLaneAI/catalyst/blob/main/frontend/catalyst/passes.py#L51

Reply:
Good catch! Added a list of all the supported gates.

Reply:
The rendering seems a bit weird https://xanaduai-pennylane--1136.com.readthedocs.build/projects/catalyst/en/1136/code/api/catalyst.passes.cancel_inverses.html ?

(In CI there's a "build the docs" that renders the documentation on a PR)

Reply:
@paul0403, How about now? https://xanaduai-pennylane--1136.com.readthedocs.build/projects/catalyst/en/1136/code/api/catalyst.passes.cancel_inverses.html